system/Router/RouteCollection.php

```diff
@@ -992,6 +992,19 @@ public function cli(string $from, $to, ?array $options = null): RouteCollectionI
         return $this;
     }
 
+    /**
+     * Specifies a route that will only display a view.
+     * Only works for GET requests.
+     */
+    public function view(string $from, string $view, ?array $options = null): RouteCollectionInterface
+    {
+        $to = static fn (...$data) => view($view, ['_page' => $data]);
```

Comment:
I don't see the point in assigning an anonymous function to a variable. We can pass it right away as an argument.
Also I'm not sure about the need to pass captured URL segments.

Reply:
The variable seems to make the code a bit easier to read.

But why the passed variable name is `_page`?

Reply:
@iRedds Good point about the variable, but I think readability trumps efficiency here. 

I think it's a must to pass URL segments as they can be used within View Cells for more flexible pages. 

@kenjis Thinking back it's probably not necessary. My original thought was I wanted to make sure it wouldn't conflict with anything the user might provide - but they wouldn't provide anything in this scenario. So removing the underscore makes sense.